reana/reana_dev/git.py

```diff
@@ -1365,7 +1371,8 @@ def git_upgrade_requirements(ctx, component, exclude_components):  # noqa: D301
         if upgrade_requirements(component):
             run_command(f"git add {PYTHON_REQUIREMENTS_FILE}", component)
             run_command(
-                'git commit -m "installation: bump all dependencies"', component
+                f'git commit -m "build(deps): bump all dependencies{get_commit_pr_suffix(component)}"',
```

Comment:
`build(python)` perhaps, so that we restrict the meaning of "all" here? (alternatively, we could use "bump all Python packages")